README.md

```diff
@@ -16,6 +16,36 @@ All models are built using the `core` package, which contains the necessary meth
 
 To know more about how to work and build data models, take a look at the [models overview](./docs/contributing/test-data-models-overview.md) first and head to the ["Create new model" guideline](./docs/guidelines/creating-new-model.md) as a follow up to learn how to implement a new one.
 
+# Ownership & Gouvernance
+
+## Ownership 
+
+> [!IMPORTANT]  
+> This repository is not owned by a single team. Each individual aspect of this repository is owned by the respective teams of their respective domains.
+
+## Contributing 
+
+It is _mandatory that before contributing to this repository_ you make sure you have read the [contribution guidelines](https://github.com/commercetools/test-data/blob/main/CONTRIBUTING.md) as well as the documentation about the [test data models overview](https://github.com/commercetools/test-data/blob/main/docs/contributing/test-data-models-overview.md) and [building a new test data model](https://github.com/commercetools/test-data/blob/main/docs/guidelines/creating-new-model.md) if needed.
+
+Each respective team is responsible for the test data models of their domain in their repository to follow the defined patterns and to use these models in their applications instead of writing custom test data models in a bespoke repository.
+
+## Code review
+
+Pull Request reviews must and is expected to only involve the team owning the respective models by their domain. It can only, on demand and on a voluntary basis, involve other team(s).
+
+## Release process
+
+Releases of test-data models can be created when needed by merging a respective "Version Packages" Pull Request. Keep in mind that releases should be aggregated whenever possible. As a result before releasing you are expected to:
+
+1. Check open Pull Requests which are pending review and contact creators if they wish to be part of the next release
+    - If code review is close to be finished help to ensure it progresses to include the change in your release
+    - If code review of Pull Request(s) requires more time inform the owner that the change won't be part of your release
+2. Perform the release and inform any other teams are have functionality in the release
```

Comment:
> ... teams are have functionality...

Sorry but I don't understand this sentence.
I guess you mean that the one trigger the release should notify the other teams who have contributions in that version, right?

Reply:
Yes, better wording. Updating.